routers/auth.py

```diff
@@ -97,10 +100,23 @@ async def forgot_password():
         return ServerError(err=e,errMsg=str(e))
     
 
-@router.post("/password/reset")
-async def reset_password():
+@router.patch("/password/reset")
+async def reset_password(request: Request):
```

Comment:
Lets name all incoming request bodies as request, but their types should be understandable

It should be `request : ResetPassword`

The types should be very clear